controllers/controllers.js

```diff
@@ -8,10 +8,26 @@ exports.getAllTopics = (req, res, next) => {
     .catch((err) => {
         next(err)
       })
+    }
+    
+exports.getArticleById = (req, res, next) => {
+    const artId = req.params.article_id
+    selectArticleById(artId)
+    .then((article) => {
+        
+        if(article[0] === undefined) {
+            res.status(404).send({ msg: "article does not exist" })
+        }
+
```

Comment:
This should really be handeled in the model.
The controller is only for passing data.